pex/finders.py

```diff
@@ -101,7 +101,8 @@ class WheelMetadata(pkg_resources.EggMetadata):
 
   @classmethod
   def _split_wheelname(cls, wheelname):
```

Comment:
It sortof looks like this method is assuming it will get 4 entries (because it later takes the last 3, which might not make sense if there weren't 4). Is it worth asserting that `len(split_wheelname) == 4`?

Reply:
sure, fixed.